translation/en.json

```
@@ -0,0 +1,49 @@
+{
+    "number_topic":"Topic number",
+    "comment_posted":"Comment Posted ✅",
+    "first_link_your_account":"⛔ You need to link your account first",
+    "first_link_your_account_matrix":"⛔ You need to link your account, by sending 'discourse' or '6'",
+    "writer":"🚻 Author",
+    "date":"date",
+    "publication_time":"publish date",
+    "err_active_in_the_chat":"🔴 the Bot is already active in this chat",
+    "active_bot":"Bot activated ✅",
+    "admin_activate":"⛔ You need to be an Admin to activate this Bot",
+    "category_id":"Send the Category's 🆔 you want to subscribe to",
+    "category_id_all":"If you want to receive posts from all categories type '0'",
+    "id":"Identifier",
+    "err_wrong_entry":"Wrong entry",
+    "send_category_id":"Send the Category's 🆔",
+    "topic_title":"🔤 Topic title:",
+    "topic_content":"🔤 Topic content:",
```

In Arabic translation is
```json
"topic_title":"قم بكتابة عنوان الموضوع",
"topic_content":"قم بكتابة محتوى الموضوع"
```
So I think it's should be:
```suggestion
    "topic_title":"🔤 Send me the topic title",
    "topic_content":"🔤 Send me the topic content",
```